test/support/acceptance_test.rb

```diff
@@ -77,9 +77,12 @@ def assert_speedup(ratio = DEFAULT_SPEEDUP)
 
       def without_gem(name)
         gem_home = app.gem_home.join('gems')
+        spec_home = app.gem_home.join('specifications')
         FileUtils.mv(gem_home.join(name), app.root)
+        FileUtils.mv(spec_home.join("#{name}.gemspec"), app.root)
```

Comment:
Without moving the spec file, `gem "spring", spring.version` succeeds even though the gem code isn't there. The the next line raises a `LoadError` instead of a `Gem::LoadError`. That doesn't really matter for this solution but was an issue when trying to rescue only the `Gem::LoadError`. Anyway, I thought that this is more realistic for future specs.